### ISSUE TITLE
How to set a custom root of the local development environment to be synced?

### ISSUE THREAD

**Issue by nikolaynikolaevn** [liximomo/vscode-sftp#1004]
**Is this a similar or duplicate feature request?**
- [ ] I don't know. I will go check it.
- [x] No.

**Is your feature request related to a problem? Please describe.**

I am having a project where my local development environment has a different folder structure from my remote environment, e.g. the project files are located at html/ at the remote, but at "public_html/" locally. Is there a way to set/map the local root source code directory to the remote root source code directory?

**Describe the solution you'd like**
A functionality for setting the root source folder to be synced to the remote and/or creating mappings of local->remote directories to sync.

**Describe alternatives you've considered**

**Does this project help you?**
- [x] Yes. SFTP IS AWESOME!
- [ ] Yes. [I want to support you.](https://github.com/liximomo/vscode-sftp#donation)

**Comment by sylafrs** [comment]
Yup, to me this is a required feature, I can't really use this without; so I'll remove the extension ^^'
